ext/net/ops.rs

```diff
@@ -165,7 +169,7 @@ pub(crate) fn accept_err(e: std::io::Error) -> NetError {
 pub async fn op_net_accept_tcp(
   state: Rc<RefCell<OpState>>,
   #[smi] rid: ResourceId,
-) -> Result<(ResourceId, IpAddr, IpAddr), NetError> {
+) -> Result<(ResourceId, IpAddr, IpAddr, Fd), NetError> {
```

Comment:
If Fd is not available on windows, maybe it's better to do:

```suggestion
) -> Result<(ResourceId, IpAddr, IpAddr, Option<Fd>), NetError> {
```